index.html

```diff
@@ -307,7 +318,42 @@ <h3 class="contact-me">I'm always interested in hearing about new projects, so i
     </footer>
 
 </section>
+</div>
+<div class="modal" id="modal">
+    <div class="modal-header">
+        <div>
+            <h1 class="mod-header">Multi Post Stories</h1>
+            <h1 class="mod-header1">Keeping track of hundreds  of components website</h1>
+        </div>
+    <div>
+            <button data-close-button id="mod-button" class="fa-solid fa-xmark" ></button>
+    </div>
+    </div>
+    <div class="mod-btn-container">
+    <ul class="mod-buttons">
+        <li><button class="mod-btn" type="button">HTML</button></li>
+        <li><button class="mod-btn" type="button">Bootstrap</button></li>
+        <li><button class="mod-btn" type="button">Ruby on Rails</button></li>
+    </ul>
+    </div>
+    <div class="images-popup">
+        <img src="images/Snapshoot-Portfolio.png" alt="IOT-10-image" class="mod-img">
+        <img src="images/Snapshoot-Portfolio1.png" alt="IOT-10-image2" class="mod-img1">
+        <div class="mod-para"><p class="first">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.<br><br>
 
+        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.</p>
```

Comment:
- [ ] [OPTIONAL] For Smaller Desktop screens of width 1300px, your modal is too big for them. Please kindly make use of percentages for your width so that your modals are responsive enough

<img width="960" alt="image" src="https://user-images.githubusercontent.com/52934128/231572381-fd4fc176-3c84-4ba0-94f2-70acac32659c.png">